src/components/Table.jsx

```diff
@@ -96,18 +96,18 @@ class Table extends React.Component {
 }
 
 const mapStateToProps = state => ({
-  cells: state.move.cells,
-  move: state.move.move,
-  indexX: state.setIndex.indexX,
-  indexO: state.setIndex.indexO,
-  moveInfo: state.move.info,
-  count: state.move.count,
+  cells: state.cells,
+  move: state.move,
+  indexX: state.indexX,
+  indexO: state.indexO,
+  moveInfo: state.info,
+  count: state.count,
```

Comment:
you could create a selector to return an object with all the fields you need
instead of getting them one by one